### ISSUE TITLE
Feature request: Allow to provide the number of commits to display on each release.

### ISSUE THREAD

**Issue by M4RC3L05** [sunesimonsen/offline-github-changelog#101]
Hi,

Based on some requirements of mine, I would like to know if allowing to specify the number of commits to display on each release block would be something that this package could have.

If so, i would suggest something like a flag `commits` we can pass to the command:

```bash
$ offline-github-changelog --commits 10 > CHANGELOG.md
```
Also, this flag could also receive `-1` as a way to include all commits (for some testes i did we could map `-1` to Number.POSITIVE_INFINITY).

If this is something you see value on adding i can start drafting the changes.

**Comment by papandreou** [comment]
What would happen if there are more commits than the specified number?

**Comment by M4RC3L05** [comment]
> What would happen if there are more commits than the specified number?

I would argue that it should continue to behave the same as it does now, with the `[+7 more]()`.

My idea is to control the `MAX_REGULAR_COMMITS_PER_RELEASE` to allow it to be a value other 5

**Comment by sunesimonsen** [comment]
@M4RC3L05 could you maybe try to edit [this](https://github.com/sunesimonsen/offline-github-changelog/blob/master/CHANGELOG.md) changelog to show us what you intent happening when `--commits 3` is passed as a parameter.

**Comment by sunesimonsen** [comment]
Why show only some of the commits - I can understand leaving the commit list out, or truncate it completely into a range with a count.

**Comment by M4RC3L05** [comment]
I only suggested the commits number flag to be feature complete, because, for my use case I actually want to dump all of the commits between releases.

Example: https://raw.githubusercontent.com/M4RC3L05/music-follower/main/CHANGELOG.md

Currently I am able to get all the commits without them being truncated past 5 with this [patch](https://raw.githubusercontent.com/M4RC3L05/music-follower/main/patches/offline-github-changelog%2B3.0.1.patch), where I override `MAX_REGULAR_COMMITS_PER_RELEASE` to be `Number.POSITIVE_INFINITY`

**Comment by sunesimonsen** [comment]
@M4RC3L05 I don't think what you describe is something other people would use. But I could agree on having a `--truncate-commits` flag, that truncates the commits with a number to the range of commits.

**Comment by M4RC3L05** [comment]
That can work for mw to tbh. just need to pass a big value to `--truncate-commits` that way i am able to show them all.

If it is ok by you i think i could start drafting something?

**Comment by sunesimonsen** [comment]
I just realised that we already have [some kind of control over the number of commits](https://github.com/unexpectedjs/unexpected/blob/master/CHANGELOG.md#commits-to-master-3) - sorry it has been a while since I worked in this code base.

I now understand what you want to do - sorry for misunderstanding it. I think it makes sense to be able to control `MAX_REGULAR_COMMITS_PER_RELEASE` with a command line argument. 

So I think your initial idea probably holds up just fine. I wasn't really aware that we already did something like this.
